### DISCUSSION TITLE
Welcome to rails_hotwire_base Discussions!

### DISCUSSION THREAD

**Discussion by brunvez** [rootstrap/rails_hotwire_base#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by brunvez** [comment]
@santib @JuannFerrari anyone else seeing these warnings? If so we should create a ticket and address them

![image](https://user-images.githubusercontent.com/17035407/104827856-683ed980-5841-11eb-9fd8-788543dbe61e.png)

**Reply by brunvez** [reply]
(while running `be rspec`)

**Reply by santib** [reply]
yep I'm seeing them. Seems that they were introduced by Rails 6.1 (I tried with Rails 6.0 and weren't happening). Not sure if we can fix it on our end or if we need Rails 6.1 to get them fixed (we can make a PR to Rails maybe)
